platformio.ini

```diff
@@ -110,7 +110,9 @@ lib_ignore =
 extra_scripts =
     pre:./scripts/add_os_specific_build_flags.py
     pre:./scripts/create_webots_library.py
+    pre:./scripts/copy_sounds.py

```

Comment:
Could we integrate it in the other script and rename the export_executable.py to something like package.py ?

Reply:
I do not think this is a good idea, as these are two separate funcitonalities. The sounds must be copied in order for the executable to work. If both are in the same script, and at some point the ZIP file functionality is to be removed, the sounds would not be copied and we will have and error. For this reason I separated the packing and the copying of sounds from each other.

Reply:
Hmm ... but there is no need to copy it to .pio/build/.... because if you run it from VSCode, its fine.
I thought its only necessary to package it there? Lets talk about in detail.

Reply:
But if you run it using the Webots launcher, then it is not right anymore. And it does not work for the Webots launcher if it is in a Zip file.